{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -0,0 +1,56 @@
+PYTHON_VERSION := 3.11
+WHEEL_VERSION := 0.43.0
+
+.DEFAULT_GOAL := venv/bin/activate
+
+venv/bin/activate: requirements-dev.txt
+		python$(PYTHON_VERSION) -m venv venv
+		. venv/bin/activate
+		pip install --upgrade pip
```

Comment:
I am getting following output:

```
~/Documents/Projects/ds-template/{{ cookiecutter.repo_name }}$ make 
python3.11 -m venv venv
. venv/bin/activate
pip install --upgrade pip
make: pip: No such file or directory
make: *** [Makefile:9: venv/bin/activate] Error 127
```

Reply:
Does the error still occur if you run make after generating the test project? Can you check if pip is configured correctly on your machine?